client/reader/sidebar/index.jsx

```diff
@@ -36,8 +37,12 @@ import config from 'config';
 import userSettings from 'lib/user-settings';
 import AppPromo from 'components/app-promo';
 import { setNextLayoutFocus } from 'state/ui/layout-focus/actions';
+import userUtils from 'lib/user/utils';
+import * as userAgentUtils from 'lib/user-agent-utils';
```

Comment:
I was getting a weird error with a previous version of the code by importing just isChromeOS... I'd have to rollback a few commits and see but I can change it as the structure is different now.